src/components/dashboard/main-content/course-enrollments/data/tests/hooks.test.jsx

```diff
@@ -927,12 +931,14 @@ describe('useContentAssignments', () => {
   }) => {
     const mockEnrollment = {
       ...mockTransformedMockCourseEnrollment,
+      courseKey: 'edX+DemoX',
       mode,
       enrollBy,
+      isAssignedCourseRun: true,
```

Comment:
I believe this variable should represent a raw course enrollment object, not an assignment transformed to have the data structure of an enrollment. While a raw enrollment will have `course_key`, I don't believe it would have `isAssignedCourseRun`. 

Consider that the existing enrollment in this use case likely was not originated via an assignment (e.g., an audit enrollment created via the b2c marketing website).

I also might recommend adding the `courseKey` attribute via the `createRawCourseEnrollment` test utility since it should generally exist for all enrollments.